e2e/page-objects/toolBar.ts

```diff
@@ -105,6 +105,21 @@ export class ToolBar extends HelperBase {
     await tooltip.waitFor({ state: 'hidden' });
   }
 
+  /**
+   * Hover over element and verify that tooltip does not appear
+   * @param element - playwright locator for the element
+   * @param tooltipText - text of the tooltip
+   */
+  @step(
+    (args) =>
+      `Hover over the element and ensure tooltip with text '${args[1]}' does not appear`,
+  )
+  async hoverElAndCheckNoTooltip(element: Locator, tooltipText: string) {
+    await element.hover();
```

Comment:
before hovering element it is needed to expect this element to be visible (wait to be visible in playwright)